app/pages/admin/project-status.jsx

```diff
@@ -185,6 +192,46 @@ class ProjectStatus extends Component {
     return (
       <div className="project-status">
         <ProjectIcon project={this.state.project} />
+
+        <div className="project-status__section project-status__section--state">
+          <h4>Project State</h4>
+          <fieldset>
+            <label htmlFor="project-state-active">
```

Comment:
Unfortunately this wasn't setup with active in the enum states api side. I apologise for this. The enum states you can set are paused and finished only. Thus the active setting here acts as a reset instead of an actual state. I would assume clicking active would persist that change through the UI. Perhaps we should rename active to reflect this behaviour. 

The state param is a ***** mess api side, if the enum field is set it uses that if not it proxies to the live setting. I have no idea why but it does.

Reply:
Right - it now sets it to an empty string instead of `live`